src/sql/src/normalize.rs

```diff
@@ -502,7 +502,7 @@ macro_rules! generate_extracted_config {
         paste::paste! {
             #[derive(Debug)]
             pub struct [<$option_ty Extracted>] {
-                seen: HashSet::<[<$option_ty Name>]>,
+                seen: ::std::collections::BTreeSet::<[<$option_ty Name>]>,
```

Comment:
Using the fully qualified path for types used in macros ensures that callers don't need to have these types already in scope.